bam_fastq.c

```diff
@@ -64,8 +64,12 @@ static void bam2fq_usage(FILE *to, const char *command)
 "  -o FILE      write reads designated READ1 or READ2 to FILE\n"
 "               note: if a singleton file is specified with -s, only\n"
 "               paired reads will be written to the -1 and -2 files.\n"
-"  -f INT       only include reads with all  of the FLAGs in INT present [0]\n"       //   F&x == x
-"  -F INT       only include reads with none of the FLAGS in INT present [0x900]\n"       //   F&x == 0
```

Comment:
added longoption arguments for filtering flags for clarity, consistent with [samtools view](https://github.com/samtools/samtools/blob/develop/sam_view.c#L1471).